lib/linguist/languages.yml

```diff
@@ -3919,6 +3919,14 @@ Linux Kernel Module:
   tm_scope: none
   ace_mode: text
   language_id: 203
+Liquidsoap:
+  type: programming
+  color: "#990066"
+  extensions:
+  - ".liq"
+  tm_scope: source.liquidsoap
+  ace_mode: text
+  language_id: 614641732
```

Comment:
`Liquidsoap` needs to be sorted _after_ `Liquid:`, not before (as per [alphabetisation requirements](https://github.com/github-linguist/linguist/blob/39fd5e93de98de3434cb3e857c5b1972c418f8c5/CONTRIBUTING.md)).

Reply:
Done.